invenio_accounts/sessions.py

```diff
@@ -52,6 +52,9 @@ def add_session(session=None):
         is used. The object is expected to have a dictionary entry named
         ``"_user_id"`` and a field ``sid_s``
     """
+    if '_user_id' not in session and 'user_id' in session:
+        session['_user_id'] = session['user_id']
```

Comment:
not sure if this is just symptom treatment, but it fixes the `KeyError` that i've posted in Discord for me